### PR TITLE
Update agenda with Learn unsafe Rust

### PR DESCRIPTION
**Pull request by PLeVasseur**
Add unsafe Rust coding considerations

### CODE REVIEW

**Review by jmqd — APPROVED** [review]
LGTM, added minor typo/spell fixes.

File: subcommittee/coding-guidelines/meetings/19-November-2024/agenda.md
Comment (ah801) [inline subcommittee/coding-guidelines/meetings/19-November-2024/agenda.md:1]: As these meetings are related to safety, I think we should be recording the meetings for audit and traceability purposes as well as making it easier to review the decision trails. I'm not sure if they are already being recorded, if they are then ignore the rest.
If they are not then I suggest we introduce a new item (0) on the Agenda, to decide whether or not to record going forward. We can take a show of digital hands for and against. I noticed in google meet if you select either people or chat (I can't remember which) you can see a list of who has voted to make it easier to count (not sure if the host can see this).
The host can chose to record or not going forward (including the current meeting) based off this decision.

We can subsequently decide whether or not to get an AI to transcribe the recording (or even the live meeting) as an additional resource (only).

Comment (PLeVasseur) [reply subcommittee/coding-guidelines/meetings/19-November-2024/agenda.md:1]: Hey @ah801 -- I like the idea of bringing this to the group. Can you submit a suggestion to the PR adding an item 0. for this and I'll commit the suggestion?

**Review by ah801 — CHANGES_REQUESTED** [review]
As these meetings are related to safety, I think we should be recording the meetings for audit and traceability purposes as well as making it easier to review the decision trails.

I suggest we introduce a new item (0) on the Agenda (or shift all down):

(0) Decide whether or not to record the meetings going forward (video and audio)

We can subsequently decide whether or not to get an AI to transcribe the recording (or even the live meeting) as an additional resource (only).

File: subcommittee/coding-guidelines/meetings/19-November-2024/agenda.md
Comment (PLeVasseur) [reply subcommittee/coding-guidelines/meetings/19-November-2024/agenda.md:1]: Okay! I added this.

**Comment by JoelMarcey** [comment]
I couldn't attend the meeting today, but I think we should be careful about a blanket statement about recording meetings. Besides the logistics of actually maintaining and posting the recording (which are sometimes not trivial), there could be potential privacy concerns as well, particularly if someone says something that in hindsight should be off the record, which does happen. I am not saying never record, but I think we need to consider the implications before making it the default process for all meetings.

The written minutes should serve as the authoritative record for meetings, imho.

**Comment by ah801** [comment]
Hi @JoelMarcey we voted to not do any recordings at this time for similar reasons.

**Comment by JoelMarcey** [comment]
Ok. Thanks. I guess if I would have looked at the authoritative minutes, I would have found that out 😂

**Pull request merged**